src/promptflow/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -231,13 +231,29 @@ def __post_init__(self, exception, status_code):
 
 
 def build_pfs_user_agent():
+    # For local pfs:
+    # 1. the operation context won't be cleared after each request.
```

Comment:
Could you add more description on why we need to do this?

I think OperationContext storage is leveraging ContextVar, which is thread local.
Is the requirement comes when new request need to refresh operationConetxt?

Reply:
You mean that execution should be done in different thread, so no need to do this clear-up? This is mainly for operations like run.list and connection.list, when we won't start a new thread for them.